client/src/pages/Review/Review.tsx

```diff
@@ -86,7 +86,7 @@ const Review = () => {
       formData.append("outline_file", file as File);
 
       await axios
-        .post("/files", formData, {
+        .post("/premium-files", formData, {
```

Comment:
So we'll use the endpoint powered by ChatGPT as the only one till we have the front-end to support both?